.pre-commit-config.yaml

```diff
@@ -197,16 +193,8 @@ repos:
 
             # builtin filter function
             |(?<!def)[\(\s]filter\(
-
-            # exec
-            |[^a-zA-Z0-9_]exec\(
         types_or: [python, cython, rst]
         exclude: ^doc/source/development/code_style\.rst  # contains examples of patterns to avoid
-    -   id: cython-casting
```

Comment:
is this one covered by anything else?

Reply:
This one is in `id: unwanted-patterns-in-cython` (would be nice if this was in cython-lint!)